util/collections/src/garbage/mod.rs

```diff
@@ -0,0 +1,20 @@
+pub mod atomic;
+pub mod counted;
+pub mod map;
+pub mod set;
+use std::num::NonZeroU64;
+
+#[derive(Debug, Clone)]
+pub struct Duration(NonZeroU64);
```

Comment:
Does `garbage::Duration` provide added value over a more widely used domain type like `std::time::Duration`?

Reply:
It is specifically to restrict the domain to non-zero while remaining a very simple `u64` wrapper.